### PR TITLE
Videos Index Controller Action and Tests

### PR DESCRIPTION
**Pull request by dnguye2**

### CODE REVIEW

File: test/controllers/videos_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      expect(video).must_be_instance_of Hash
+      expect(video.keys.sort).must_equal VIDEO_FIELDS
+    end
+  end
+
+  it "returns an empty array if no videos exist" do
+    Video.destroy_all
+
+    get videos_path
+
```
Comment (theomoondev) [inline test/controllers/videos_controller_test.rb:30]: Should we add this line here to check that it responds with status 200 when there are no videos? 

```suggestion
must_respond_with :ok

```

**Review by theomoondev — APPROVED** [review]
Looks great! I just have one comment about a test (see below... or above... I'm not sure where they will show up haha)

File: test/controllers/videos_controller_test.rb (hunk 1 above)
Comment (dnguye2) [reply test/controllers/videos_controller_test.rb:30]: Agree! I have it in line 33 but am not sure the difference between using `:success` and `:ok`. It seems to both pass the test?

Comment (theomoondev) [reply test/controllers/videos_controller_test.rb:30]: Ahh I didn't look close enough, sorry! I think they both check for 200 so this looks great!

**Pull request merged**